src/app/services/ccd-config/ccd-case.config.ts

```diff
@@ -148,4 +148,12 @@ export class AppConfig extends AbstractAppConfig {
   public getWorkAllocationApiUrl(): string {
     return this.workallocationUrl;
   }
+
+  public getHrsUrl(): string {
+    return '';
+  }
+
+  public getRemoteHrsUrl(): string {
+    return '';
+  }
```

Comment:
DON'T approve and merge until you know these two exact values, this is just for get a build pipeline run successfully and release to preview for service testing.